gremlin-dotnet/src/Gremlin.Net/Driver/Connection.cs

```diff
@@ -68,18 +71,54 @@ public async Task ConnectAsync()
 
         public async Task CloseAsync()
         {
+            if (!String.IsNullOrEmpty(_sessionId))
+            {
+                await closeSession().ConfigureAwait(false);
+            }
             await _webSocketConnection.CloseAsync().ConfigureAwait(false);
         }
 
+        private async Task closeSession()
+        {
+            var msgBuilder = RequestMessage.Build(Tokens.OpsClose).Processor(Tokens.ProcessorSession);
+            msgBuilder.AddArgument(Tokens.ArgsGremlin, "session.close()").AddArgument("session", _sessionId);
```

Comment:
Please add a new token like `Tokens.ArgsSession` here instead of hard-coding the string here to stay consistent.

Reply:
Also, is the `gremlin` argument really necessary here? I don't see it being used in [the docs of session close](http://tinkerpop.apache.org/docs/3.4.6/dev/io/#_session_close_2). If it's needed, then I would suggest that we add the value also as a token maybe as something like `Tokens.DummyValue` with a string value and XML comment making it clear that the value doesn't matter.

Reply:
thanks, I add "session().close()" as tips for debug things,  the processor will ignore it. I will add comment for this.